src/SignalR/clients/ts/signalr/src/Errors.ts

```diff
@@ -65,3 +67,116 @@ export class AbortError extends Error {
         this.__proto__ = trueProto;
     }
 }
+
+/** Error thrown when the selected transport is unsupported by the browser. */
+export class UnsupportedTransportError extends Error {
```

Comment:
It might be a good idea to throw
```
/** @private */
```

on all these, I believe it helps hide the classes from our API docs. Example https://github.com/dotnet/aspnetcore/blob/47be8bc8eb52f63f0ba82b2b6620988dce748766/src/SignalR/clients/ts/signalr/src/TextMessageFormat.ts#L5